source/adminguide/storage.rst

```diff
@@ -169,6 +174,13 @@ In order to use multiple local storages, you need to
 
 #. Restart cloudstack-agent service
 
+Adding Local Storage Pool via UI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When using UI, ensure that the scope of the storage is set to "Host", and 
+ensure that the protocol is set to "Filesystem" 
+
+|adding-local-pool-via-ui.png|
```

Comment:
@andrijapanicsb 
the following is required to be at the end of the file. otherwise the image is not shown

```
.. |adding-local-pool-via-ui.png| image:: /_static/images/adding-local-pool-via-ui.png
   :alt: Adding Local Storage Pool via UI
```

Reply:
@andrijapanicsb ^^